packages/cli/templates/default-plugin/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "@backstage/cli/config/tsconfig.json",
+  "include": [
+    "src",
+    "dev",
+    "migrations"
```

Comment:
No need to have this for frontend plugins

Reply:
done